.github/workflows/reddit.yml

```diff
@@ -17,8 +17,7 @@ jobs:
           app-secret: ${{ secrets.REDDIT_APP_SECRET }}
           #subreddit: ferdium # Submit to /r/ferdium # uncomment this after testing on a dummy subreddit when moving to the main one
           subreddit: ferdiumtests # Submit to /r/ferdiumtests # Delete this line after testing on dummy repository
-          title: [Release] Ferdium ${{ github.event.release.tag_name }}
-          url: ${{ github.event.release.html_url }}
```

Comment:
do we need to remove this line? It defaults to that, nevertheless, I think

Reply:
yes, since that's the default, i removed it in the PR